### ISSUE TITLE
fail to start "local-cluster" on windows for docker

### ISSUE THREAD

**Issue by askme765cs** [Segence/docker-hadoop#4]
`root@hadoop-namenode:~# service hadoop start

====== STARTING HDFS ======

Starting namenodes on [hadoop-namenode]
hadoop-namenode: Warning: Permanently added '[hadoop-namenode]:2222,[172.18.0.2]:2222' (ECDSA) to the list of known hosts.
hadoop-namenode: starting namenode, logging to /usr/local/hadoop/logs/hadoop-hadoop-namenode-hadoop-namenode.out
: Temporary failure in name resolutionop-datanode1`

**Comment by saifjsl** [comment]
It's failing for me also on the same stage, although my error is different:
```: name or service not known  Hadoop-datanode1```

**Comment by robvadai** [comment]
Hi @saif-ellafi ,

was the EOL issue after you manually edited the file? The Git version should have UNIX EOL...

**Comment by saif-ellafi** [comment]
Yes, it's been some long time now but I had to fix the EOL manually for it to work. It was a nightmare to discover that was the actual issue though. Not even a slight clue.

**Comment by robvadai** [comment]
Shall we close this issue?
